.dockerignore

```diff
@@ -0,0 +1,14 @@
+*.md
+.bundle
+.dockerignore
+.env
+.git
+.travis.yml
+Dockerfile
+spec
+spec_handwork
```

Comment:
There is no such folder spec_handwork in repo - so it does not have to be in .dockerignore
Same with other files .bundle, DS_Store
Please double check this list